src/api/routes/rosetta/construction.ts

```diff
@@ -248,12 +275,25 @@ export function createRosettaConstructionRouter(db: DataStore, chainId: ChainID)
       recentBlockHash = blockQuery.result.block_hash;
     }
 
+    // Getting fee info
+    const feeInfo = await new StacksCoreRpcClient().getEstimatedTransferFee();
+    const currency: RosettaCurrency = {
+      symbol: RosettaConstants.symbol,
+      decimals: RosettaConstants.decimals,
+    };
+
+    const fee: RosettaAmount = {
+      value: (+feeInfo * txSize).toString(),
```

Comment:
Might this need to use a safe math lib?

Reply:
Yeah I am not sure about that. Typescript should avoid problems but not quite sure.

Will blockstack allow negative fee ? I can test for that. There is also a `max_fee` value that is used to verify we won't spend too much on fee.

Also another question, does transfer transaction have a fix size ?

Reply:
TypeScript doesn't change anything at runtime, so this could be prone to floating point imprecision 0.1 * 0.2

Negative fee should definitely fail.

I believe transfers txs do have a fixed size, but only because the `@stacks/transaction` lib pads the memo field with empty bytes

Reply:
Yeah, we should just use native `BigInt` here.